### ISSUE TITLE
Interface spécifique a un taxon (Liparis)

### ISSUE THREAD

**Issue by jpm-cbna** [PnX-SI/gn_module_flore_prioritaire#42]
Le taxon Liparis avait dans la version 1 du module une interface de saisie très proche de l'interface standard mais avec 2 champs supplémentaires au niveau du formulaire d'une ZP.

Nous proposons éventuellement d'ajouter cette interface si possible sous forme de "plugin" qui viendrait compléter/modifier l'interface lorsqu'on réalise une saisie lié au taxon en lien avec le "plugin".

L'interface de Liparis était seulement différente au nivau du formulaire et donc de l'affichage des informations d'une ZP. Voici une impression écran de l'interface de la v1:

![Screenshot_20230214_100847](https://user-images.githubusercontent.com/51449363/218690327-5ea7d2e3-e424-4679-bfea-6e96a95f175b.png)

Détail du premier champ "Zonage(s)":
![Screenshot_20230214_103527](https://user-images.githubusercontent.com/51449363/218696514-bd8d0702-9e3e-49cd-92d3-689506c55d59.png)

Liste valeurs du second champ "Actions de gestion":
[actions_gestion.txt](https://github.com/PnX-SI/gn_module_flore_prioritaire/files/10730914/actions_gestion.txt)

**Comment by camillemonchicourt** [comment]
Je n'avais pas souvenir de cette spécificité sur notre v1.
Peut-être une modification spécification sur l'instance du CBNA.
Dans tous les cas, je trouve ça un peu dommage d'avoir une spécificité du protocole pour un seul taxon.

**Comment by jpm-cbna** [comment]
@camillemonchicourt Oui, je suis d'accord. Je retranscrit ici les spécificité lié au projet Liparis afin d'en garder une trace car le site https://reseau-conservation-alpes-ain.fr/ devrait être clôturé prochainement...

Concernant les 2 champs de différence, il faut voir s'ils sont toujours utile et s'ils ne peuvent pas être intégré au tronc commun...

**Comment by camillemonchicourt** [comment]
Oui, je pense qu'il faut se recentrer sur le sujet du protocole, sa question scientifique et son modèle de données associé, et pas voir ça comme des champs que l'on ajoute pour chaque besoin.

**Comment by jpm-cbna** [comment]
Les données spécifiques au projet Liparis étaient stockées dans un schéma spécifique `liparis` de l'ancienne base. Nous les avons extraites et sauvegardé dans un fichier SQL `2023-11-07_import_schema_liparis_to_geonature2db.sql` disponible sur l'espace SFTP du CBNA.

Les données des champs "zonages" et "actions de gestion" ont été stockées dans l'objet Json du champ `additional_data` (`migrateOriginalInfos.liparisProject.zonages` et `migrateOriginalInfos.liparisProject.actionsGestion`) des 208 zones de prospection concernées (voir table `pr_priority_flora.t_zprospect`). 
Nous avons utilisé le script Bash d'import ''liparis' du dépôt [floresentinelle-data](https://github.com/cbn-alpin/floresentinelle-data/tree/main/liparis) pour réaliser cet import.
